man/journalctl.xml

```diff
@@ -474,8 +474,10 @@
         <term><option>-k</option></term>
         <term><option>--dmesg</option></term>
 
-        <listitem><para>Show only kernel messages. This implies <option>-b</option> and adds the match
-        <literal>_TRANSPORT=kernel</literal>.</para>
+        <listitem><para>Show only kernel messages. This adds the match <literal>_TRANSPORT=kernel</literal>.
+        This implies <option>--boot=0</option>, while the implication is overridden by specifying
+        <option>--boot=</option> with some boot specification, especially <option>--boot=all</option> will
+        disables the implication.</para>
```

Comment:
Hmm, this feels unnecessarily verbose to me. "This implies ... unless explicitly specified otherwise" sounds good enough to me?